### DISCUSSION TITLE
Download to SMB server network path hdd

### DISCUSSION THREAD

**Discussion by PwnyStark** [LivingWithHippos/unchained-android#346] [Ideas]
Hey, love the project.

I just started using a Samba server(smbv2) on my router with an external usb hdd; it would be great to be able to download to it. Or any network path really!  Dunno if this is possible, but it would be convenient.

Thanks for your time and for sharing with everyone.

**Comment by Invictaz** [comment]
Try sharing it to Aria2 or any other download manager that can access SMBv2
